test/unit/machine-learning/machine-learning.spec.ts

```diff
@@ -161,16 +159,6 @@ describe('MachineLearning', () => {
               + 'instance.');
     });
 
```

Comment:
Does it make sense to keep this test, but just update it to call `createModel()`?

Reply:
sure